testsuites/CBLTester/System_test_multiple_device/conftest.py

```diff
@@ -340,7 +340,10 @@ def params_from_base_suite_setup(request):
         "generator": generator,
         "resume_cluster": resume_cluster,
         "create_db_per_test": create_db_per_test,
-        "enable_rebalance": enable_rebalance
+        "enable_rebalance": enable_rebalance,
+        "testserver_list": testserver_list,
+        "test_name": request.node.name,
```

Comment:
why do we need this, you can create test_name inside the test_set_up method,right?

Reply:
the original definition of params_from_base_test_setup method didn't pass request as a parameter, therefore request.node.name is not recognized. I compared other conftest, params_from_base_test_setup all have request and params_from_base_suite_setup on the argument list. I am fixing the method definition and remove this test_name parameter.